src/main/scala/uk/gov/hmrc/play/audit/filters/FrontendAuditFilter.scala

```diff
@@ -142,6 +137,7 @@ trait FrontendAuditFilter extends AuditFilter {
   private def cleanQueryStringForDatastream(queryString: String): String = {
     queryString.trim match {
       case "" => "-"
+      case ":" => "-" // play 2.5 FakeRequest now parses an empty query string into a two empty string params
```

Comment:
so in an empty query param coming through as "", it comes as ":" - i.e. it puts the delim in even if key and value are empty.